packages/components/src/components/loading/dtrader-v2-loaders/contract-details-loader.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ContractCardLoader from './contract-card-loader';
+import { Skeleton, VARIANT } from '../../skeleton';
+
+const ContractDetailsLoader = () => {
+    return (
+        <div className='loading-dtrader-v2__contract-details' data-testid='dt_contract_details_loader'>
+            <ContractCardLoader />
+            <div className='skeleton-box__chart'>
+                <Skeleton />
+            </div>
+            <div className='skeleton-box__details'>
+                <Skeleton width={136} height={24} />
+                <div>
+                    {Array.from(new Array(4)).map((_, idx) => {
```

Comment:
Okay, time for my silly questions: will it work without `Array.from`?

Reply:
no, it won't. it should be either `new Array(4).fill('').map()` or `Array.from(new Array(4)).map()`